README.md

```diff
@@ -35,8 +35,8 @@ applications running in almost any environment. Here's an introductory video:
 
 	```JS
 	// Require the library and initialize the error handler
-	var errors = require('@google/cloud-errors')({
-		serviceContext: {service: 'my-service'}	// not needed on Google App Engine
+	var errorHandler = require('@google/cloud-errors').start({
```

Comment:
I think you're accidentally renaming `errors` back to `errorHandler` as part of a merge conflict resolution (in general, you can avoid these by pulling `master` frequently). Can you put these back?